### ISSUE TITLE
MQTT at the Transport Level

### ISSUE THREAD

**Issue by piperoc** [emqx/blog#767]
File: [/en/202305/a-comparison-of-iiot-protocols-mqtt-sparkplug-vs-opc-ua.md](https://www.emqx.com/en/blog/a-comparison-of-iiot-protocols-mqtt-sparkplug-vs-opc-ua)

The MQTT is shows at the _Transport Layer_ of the OSI model. I am not sure this is accurate as it makes it look like an alternative to TCP/IP. I believe MQTT to be at the _Application Layer_, and requires TCP/IP as transport.

**Comment by joey8869** [comment]
Agreed, MQTT is indeed an application layer of the OSI. However, MQTT is always used as a transport layer for other protocols like OPC UA and Sparkplug to ensure end-to-end message delivery. To me, it is doubtful that MQTT is an application layer of the OSI.
